src/main/java/com/iemr/mcts/data/agent/MctsOutboundCallDetail.java

```diff
@@ -78,8 +78,8 @@ public class MctsOutboundCallDetail {
 	private String outboundCallType;
 
 	@Expose
-	@Column(name = "CallID")  //CzentrixCallID
-	private String czentrixCallID;
+	@Column(name = "CallID")  //CallCentreID
+	private String callCentreID;
```

Comment:
@Vidyaaa24 This property naming is wrong. It is not callCentreID. It should be callId